invenio_rdm_records/records/api.py

```diff
@@ -264,6 +275,17 @@ class RDMDraft(CommonFieldsMixin, Draft):
         delete=False,
     )
 
+    media_files = FilesField(
+        key="media_files",
+        bucket_id_attr="media_bucket_id",
+        bucket_attr="media_bucket",
+        store=False,
+        dump=False,
+        file_cls=RDMMediaFileDraft,
```

Comment:
notice that the field is created by default (in contradiction to RDMRecord class - we don't precreate the field, it is done by the service)